brd-ios/breadwallet/src/Helpers/SimpleTextField.swift

```diff
@@ -140,6 +176,19 @@ class SimpleTextField: UIView, UITextFieldDelegate {
         textField.layer.borderColor = isVisible ? UIColor.kycGreen.cgColor : UIColor.kycGray1.cgColor
     }
     
+    func setEmptyErrorMessage(isFieldEmpty: Bool) {
+        errorLabel.isHidden = !isFieldEmpty
+        if isFieldEmpty {
+            textField.layer.borderColor = UIColor.red.cgColor
+        }
+    }
+    
+    @objc func togglePasswordView(_ sender: Any) {
+        showHidePasswordButton.isSelected.toggle()
```

Comment:
i don't think you need `showHidePasswordButton` as you basically have the same thing with `textField.isSecureTextEntry`.